plugins/feed.ts

```diff
@@ -154,6 +158,10 @@ export default function (userOptions?: Options) {
           const content = getDataValue(data, items.content)?.toString();
           const pageUrl = site.url(data.url, true);
           const fixedContent = fixUrls(new URL(pageUrl), content || "");
+          const imagePath = getDataValue(data, items.image);
+          const image = imagePath !== undefined
+            ? new URL(imagePath, site.url("", true)).toString()
```

Comment:
Don't need to create a URL. This is simpler:
```suggestion
            ? site.url(imagePath, true))
```

Reply:
done!